### PR TITLE
JBIDE-15714 Dynamic content assist for AngularJS expressions in html files

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

File: common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/text/TextProposal.java [hunk 1]
```diff
@@ -348,7 +345,7 @@ public boolean equals(Object obj) {
      * 
      * @return
      */
-	public boolean autoActivationContentAssistantAfterApplication() {
+	public boolean isAutoActivationContentAssistantAfterApplication() {
```
Comment (maxandersen) [inline common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/text/TextProposal.java]: why do we need to change this API to get angularjs content assist ?

File: common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/text/TextProposal.java [hunk 2]
```diff
@@ -211,9 +211,6 @@ public boolean hasImage() {
 	 * @return
 	 */
 	public int getPosition() {
-		if(position==-1 && getReplacementString()!=null) {
```
Comment (maxandersen) [inline common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/text/TextProposal.java]: was this adjustment a bug ?

Comment (alexeykazakov) [inline common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/text/TextProposal.java]: That was a dead code. The method was not used at all. But now we need this method. So I cleaned it up before starting using it.

File: common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/text/TextProposal.java (hunk 1 above)
Comment (alexeykazakov) [inline common/plugins/org.jboss.tools.common/src/org/jboss/tools/common/text/TextProposal.java]: This is another dead/not used API in TextProposal. But we need this method for angularjs now. There is no need to rename it, though.

**Pull request merged**